### PR TITLE
Xena branch

### PR DESCRIPTION
**Pull request by XenaF**
Updated code with optimized methods

### CODE REVIEW

File: JS/Airport.js [hunk 1]
```diff
@@ -1,120 +1,70 @@
...
-        let passengerPlanes = this.getPasPl();
-        let planeWithMaxCapacity = passengerPlanes[0];
-        for (let i = 0; i < passengerPlanes.length; i++) {
-            if (passengerPlanes[i].getPassengersCapacity() >     planeWithMaxCapacity.getPassengersCapacity()) {
-                planeWithMaxCapacity = passengerPlanes[i];
-            }
-        }
-        return planeWithMaxCapacity;
+        let passengerPlanes = this.getPassengerPlanes();
+        let list = passengerPlanes.sort((a, b) => {
```
Comment (VladSelikh) [inline JS/Airport.js]: You don't need `list` variable here.

File: JS/index.js [hunk 2]
```diff
@@ -1,37 +1,43 @@
...
-    console.log(`Passenger airport sorted by max speed: ${Airport.print(passengerAirport.sortByMaxSpeed())}`);
-    console.log(`Plane with max passenger capacity: ${Airport.print(passengerAirport.getPassengerPlaneWithMaxPassengersCapacity())}`);
-})();
\ No newline at end of file
+    let passengerAirport = new Airport(airport.getPassengerPlanes());
+    console.log(`Military airport sorted by max distance: ${printObjectToString(militaryAirport.sortByMaxDistance())}`);
+    console.log(`Passenger airport sorted by max speed: ${printObjectToString(passengerAirport.sortByMaxSpeed())}`);
+    console.log(`Plane with max passenger capacity: ${printObjectToString(passengerAirport.getPassengerPlaneWithMaxPassengersCapacity())}`);
+    console.log(`list of military planes with transport type: ${printObjectToString(militaryAirport.getTransportMilitaryPlanes())}`);
+    console.log(`List of Bomber planes In Military Planes: ${printObjectToString(militaryAirport.getBomberMilitaryPlanes())}`);
```
Comment (VladSelikh) [inline JS/index.js]: Why did you add these two lines?
There were no any in the initial code version.

File: JS/models/ClassificationLevel.js [hunk 3]
```diff
@@ -0,0 +1,8 @@
+const ClassificationLevel  = {
```
Comment (VladSelikh) [inline JS/models/ClassificationLevel.js]: Please use CAPITAL_LETTER syntax for constants.

File: JS/models/ExperimentalTypes.js [hunk 4]
```diff
@@ -1,8 +1,8 @@
 const ExperimentalTypes   = {
```
Comment (VladSelikh) [inline JS/models/ExperimentalTypes.js]: Please use CAPITAL_LETTER syntax for constants.

File: JS/models/MilitaryType.js [hunk 5]
```diff
@@ -0,0 +1,7 @@
+const MilitaryType = {
```
Comment (VladSelikh) [inline JS/models/MilitaryType.js]: Please use CAPITAL_LETTER syntax for constants.

File: JS/models/ClassificationLevel.js (hunk 3 above)
Comment (XenaF) [reply JS/models/ClassificationLevel.js]: Done

File: JS/models/ExperimentalTypes.js (hunk 4 above)
Comment (XenaF) [reply JS/models/ExperimentalTypes.js]: Done

File: JS/models/MilitaryType.js (hunk 5 above)
Comment (XenaF) [reply JS/models/MilitaryType.js]: Done

**Review by VladSelikh — APPROVED** [review]
